lib/server/handlers/sthGetDataHandler.js

```diff
@@ -32,6 +32,7 @@ var boom = require('boom');
 var stream = require('stream');
```

Comment:
A line in CHANGES_NEXT_RELEASE file should be added describing this change. Suggested line:

```
Fix: Content Disposition header encoding issue (#433)
```